src/Shared/Network/Helpers/AccountHelper.cs

```diff
@@ -32,4 +30,12 @@ public static void AddAccountProperties(this Packet packet, Account account)
 			packet.PutFloat(account.SelectedBarrack);
 		}
```

Comment:
Additionally, we could opt for using the `mapvisibility` data for populating all of the `HadVisited` object properties. This is why I decided on storing the map ID.